sf-docker/nvidia-jetson/docker-compose.yml

```diff
@@ -169,12 +169,17 @@ services:
 
   sf-graphql-api:
     image: ${REGISTRY}sf-jetson-graphql-api:${SF_VERSION}
+    privileged: true
```

Comment:
why privileged?

Reply:
all others are privileged, to be honest I didn't investigate deeply, usually if something is not copy/paste for all in the row, clients point this out and asks